### PR TITLE
Inline image support

### PR DESCRIPTION
**Pull request by Josvanegmond**

### CODE REVIEW

File: src/com/egeniq/widget/SpannableTextView.java [hunk 1]
```diff
@@ -64,8 +108,55 @@ public void append(int style, String text) {
...
+    }
+
+    /**
+     * Appends a DynamicDrawableSpan to the SpannableTextView. Update() must be called to update the TextView visually.
+     *
+     * @param span
+     */
+    public void append(DynamicDrawableSpan span) {
+        _markupList.add(new MarkupData(" ", span)); //The space will be styled with an icon
+        _markupText += " ";
```
Comment (dzolnai) [inline src/com/egeniq/widget/SpannableTextView.java]: why is this needed?

Comment (Josvanegmond) [inline src/com/egeniq/widget/SpannableTextView.java]: Image wont be displayed if I don't add a character as placeholder for the image. I agree it's a bit dirty. Open for suggestions.

Comment (andrei-egeniq) [inline src/com/egeniq/widget/SpannableTextView.java]: ImageSpan replaces a character with an image. 

http://flavienlaurent.com/blog/2014/01/31/spans/
